curve25519-dalek/Cargo.toml

```diff
@@ -62,7 +63,7 @@ cpufeatures = "0.2.6"
 fiat-crypto = { version = "0.2.1", default-features = false }
 
 [features]
-default = ["alloc", "precomputed-tables", "zeroize"]
+default = ["alloc", "precomputed-tables", "zeroize", "group"]
```

Comment:
Since we do not require Zeroize (from the rustCrypto elliptic_curves crate) in the traits of EncodableCurve in async-mpc, I think it is ok to have the feature Zeroize be `on` by default for Dalek. However, we can revisit this in the future if it causes issues